Xamarin.PropertyEditing/Xamarin.PropertyEditing.csproj

```diff
@@ -137,6 +137,9 @@
       <Generator>PublicResXFileCodeGenerator</Generator>
       <LastGenOutput>Resources.Designer.cs</LastGenOutput>
     </EmbeddedResource>
+    <EmbeddedResource Include="Properties\Resources.*.resx">
+      <Visible>false</Visible>
+    </EmbeddedResource>
     <EmbeddedResource Include="Resources\LocalizationResources.resx">
```

Comment:
Do both `LocalizationResources.resx` and `Resources.resx` need to be translated?